src/libclient/brushes/brushpresetmodel.cpp

```diff
@@ -766,7 +745,7 @@ bool BrushPresetTagModel::importMyPaintBrushPack(
 	}
 
 	QStringList order = orderFile.readUtf8()
-		.split(QRegularExpression{"\\s*\n\\s*"}, shim::SKIP_EMPTY_PARTS);
+		.split(QRegularExpression{"\\s*\n\\s*"}, Qt::SkipEmptyParts);
```

Comment:
Should use `compat::` instead of `Qt::`?

Reply:
Oops, yep!